sources/north-america/us/de/Delaware2012Orthophotography.geojson

```diff
@@ -7,6 +7,10 @@
         "country_code": "US",
         "type": "tms",
         "description": "This data set consists of 0.3-meter pixel resolution (approximately 1-foot), 4-band true color and near infrared (R, G, B, IR) orthoimages covering New Castle, Kent and Sussex Counties in Delaware."
```

Comment:
I'm pretty sure missing commas would be caught by the tests and build process.  
This happened because you added an "attribution" property, then you also need to add a ',' on the previous line because the "description" line is not last anymore.